bagger/src/files.py

```diff
@@ -20,6 +21,10 @@
 ALTO_KEYS = set()
 OBJECT_KEYS = set()
 
+# For bagging posters when running on AWS
+POSTER_CANDIDATES = [".mpg", ".mpeg", ".mp3", ".wav", ".mp4"]
```

Comment:
tiny optimisation: make this a set so lookups are O(1) rather than linear.
```suggestion
POSTER_CANDIDATES = {".mpg", ".mpeg", ".mp3", ".wav", ".mp4"}
```